src/app/products/product-card/product-card.component.html

```diff
@@ -15,17 +15,9 @@ <h5 class="card-title">{{ product.title }}</h5>
     </div>
 
     <div class="row">
-      <div class="col-sm-12">
-        <ul
-          class="list-inline rating-list"
-          *ngFor="let star of stars"
-          style="display: inline-block"
-        >
-          <li [ngClass]="{ selected: star <= product.rating }">
-            <i class="fa fa-star"></i>
-          </li>
-        </ul>
-      </div>
+      
+        <app-product-card-rating class="col-sm-12"></app-product-card-rating>
```

Comment:
How you transfer data to product-card-rating component? How it shows the start rating?